src/AMSlib/wf/workflow.hpp

```diff
@@ -68,8 +68,9 @@ class AMSWorkflow
    * (world_size for MPI) */
   int wSize;
 
-  /** @brief  Location of the original application data  (CPU or GPU) */
-  AMSResourceType appDataLoc;
+  /** @brief mlLoc describes the location of the ML model during simulation (CPU or GPU) */
+  AMSResourceType mlLoc;
+  AMSResourceType phLoc;
```

Comment:
Could you please add a "brief" for the phLoc? Thank you!

Reply:
Done.